new-site/src/components/Hero.js

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import { withPrefix } from 'gatsby';
+import { Koros } from 'hds-react';
+import PropTypes from 'prop-types';
+
+import './Hero.scss';
+
+const Hero = ({ children, backgroundImageUrl, title, text }) => {
+  return (
+    <div className="hero-container">
+      <div className="hero-wrapper">
+        <div className="hero">
+          <div className="hero-content">
+            <div className="hero-content-text">
+              <div className="hero-content-shape" />
+              <h1 className="hero-title">{title}</h1>
+              <p className="hero-text">{text}</p>
+              {children}
+            </div>
+          </div>
+          <Koros className="hero-koros hero-koros-rotated" flipHorizontal rotate="45deg" />
+          <Koros className="hero-koros hero-koros-horizontal" flipHorizontal />
+        </div>
+      </div>
+      <div className="hero-bg-image" style={{ backgroundImage: `url(${withPrefix(backgroundImageUrl)})` }} />
```

Comment:
Probably not needed for release, but I was wondering if we could/should use a different versions of the image for different screen sizes? I think they have done something like that at varhaiskasvatus: https://www.hel.fi/fi/kasvatus-ja-koulutus/varhaiskasvatus